data/notices.json

```diff
@@ -1,5 +1,17 @@
 {
   "notices": [
+    {
+      "title": "aws-iam: IAM OIDC provider used with EKS retrieves expiring certificate",
+      "issueNumber": 8607,
+      "overview": "The IAM OIDC provider used with EKS retrieves leaf certificate instead of root certificate which has a longer expiration date",
```

Comment:
When `OpenIdConnectProvider` from the `aws-iam` module is given a `url`prop of

```cluster.clusterOpenIdConnectIssuerUrl```,

where `cluster` is an EKS Cluster, a leaf certificate is retrieved where a root certificate should be retrieved instead. This means that any `OpenIdConnectProvider` used in this way will break by the end of the year, when the EKS team rotates the leaf certificates.

Reply:
I see, I did not explain that much since other notices were brief as well. Do we defer to the GH Issue for more explanations?

Let me know what is preferred here.

Reply:
we should defer to the github issue. The notice should be tell the user that there is an action they may need to take, so maybe rephrase it to:

```An issue in the `aws-iam` module's `OpenIdConnectProvider` may break your application by the end of the year. Please see the issue thread for a workaround if you cannot upgrade when the fix is released."

Reply:
I see, ok will make the update.